src/SpoolmanManager.cpp

```
@@ -1244,6 +1244,63 @@ bool SpoolmanManager::lookupSpoolByUid(const char* uid, SpoolDetails& outDetails
     return ok;
 }
 
+float SpoolmanManager::deductFromSpoolman(const char* uid, float grams) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)